client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1830,6 +1830,7 @@
   "messages.confirm-delete-vaccine-courses_one": "This will permanently remove this vaccine course",
   "messages.confirm-delete-vaccine-courses_other": "This will permanently remove the {{count}} selected vaccine courses",
   "messages.confirm-early-vaccination": "The minimum recommended date for this vaccine is {{ date }} (based on patient age and the two-dose interval). Are you sure you can administer this vaccine?",
+  "messages.confirm-skip-dose": "You are about to administer a dose even though earlier doses were not given. This will make earlier doses no longer editable. Are you sure you want to continue?",
```

Comment:
For me earlier doses are still editable? Is this a setting or is the message wrong?